tests/utils/v2Data/events.js

```diff
@@ -0,0 +1,310 @@
+const uuid = require('uuid');
+const { UtapiMetric } = require('../../../libV2/models');
+
+const { range, randChoice, randInt } = require('./rand');
+
+const requiredFields = {
+    uuid: () => uuid.v4(),
+    account: params => params.account,
+    location: params => params.location || 'ring',
+};
+
+const objOpFields = {
+    user: params => params.user,
+    bucket: params => params.bucket,
+    object: params => params.object,
+};
+
+const eventTemplates = {
+    get: {
+        ...objOpFields,
+        outgoingBytes: params => params.currentSize,
+    },
+    put: {
+        ...objOpFields,
+        incomingBytes: params => params.newSize,
+        sizeDelta: params => {
+            if (params.versioning) {
+                return params.newSize;
+            }
+            return params.newSize - params.currentSize;
+        },
+        objectDelta: params => {
+            if (params.versioning) {
+                return 1;
+            }
+            return params.exists ? 0 : 1;
+        },
+    },
+    delete: {
+        ...objOpFields,
+        sizeDelta: params => {
+            if (params.versioning) {
+                return 0;
+            }
+            return Math.floor((params.currentSize / params.versions)) * -1;
```

Comment:
minor readability hint: writing `-1 * Math.floor(...)` to emphasize the multiplicator